openfisca_aotearoa/variables/acts/super/super.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and entitlement system
+from openfisca_aotearoa.entities import Person
+
+
+class super__living_alone(Variable):
+    value_type = int
+    entity = Person
+    definition_period = MONTH
+    label = u"Is a person considered as living alone"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/whole.html#DLM5578822"
+
+
+class super__has_partner_in_long_term_care_or_rest_home(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Has a partner in long term care or rest home"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/DLM114223.html"
+
+
+class super__eligibility(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Classified as eligible for NZ Super"
+
+    def formula(persons, period, parameters):
+        return (persons('age', period) >= 65) *\
+            not_(persons('acc__is_receiving_compensation', period)) +\
+            persons('veterans_support__is_entitled_to_be_paid_veterans_pension', period)
```

Comment:
Is this variable returning true if the person gets the normal pension or the veterans pension?